compiler/src/utils.ml

```diff
@@ -394,6 +394,45 @@ let hierror ~loc ?funname ~kind ?sub_kind ?(internal=false) =
       raise (HiError err))
 
 
+(* -------------------------------------------------------------------- *)
+(** Splits a time in seconds into hours, minutes, seconds, and centiseconds.
+  Number of hours must be below one hundred. *)
+let hmsc f =
```

Comment:
This look like localtime, the diff is in the c?
Maybe also in no day but only our.

Reply:
There is some timezone for with a localtime of 1s is 11:00pm. This is not what I want here…